lib/phx_diff_web/live/compare_live.ex

```diff
@@ -65,4 +66,16 @@ defmodule PhxDiffWeb.CompareLive do
   defp page_title(%AppSpecification{} = source, %AppSpecification{} = target) do
     "v#{source.phoenix_version} to v#{target.phoenix_version}"
   end
+
+  @github_url "https://github.com/phoenix-diff/phoenix-diff/tree/master/priv/data/sample-app/"
+  defp target_github_url(%{phx_new_arguments: arguments, phoenix_version: version}) do
+    @github_url <> Path.join(to_string(version), phx_argument_path(arguments))
+  end
+
+  defp phx_argument_path(["--no-html"]), do: "no-html"
+  defp phx_argument_path(["--no-ecto"]), do: "no-ecto"
+  defp phx_argument_path(["--no-live"]), do: "no-live"
+  defp phx_argument_path(["--binary-id"]), do: "binary-id"
+  defp phx_argument_path(["--umbrella"]), do: "umbrella"
+  defp phx_argument_path(_arguments), do: "default"
```

Comment:
You should be able to replace this with `PhxNewArgListPresets.preset_from_arg_list/1` and then calling `.id` on the result.

https://github.com/phoenix-diff/phoenix-diff/blob/6c66685c61296f419af86f4150a3f51719f813f3/lib/phx_diff_web/diff_selections/diff_selection/phx_new_arg_list_presets.ex#L49-L54

Reply:
The id still needs some massaging to replace "_" with "-" for example. How about adding a `:path` field to `PhxNewArgListPreset`?